packages/feedback/src/constants.ts

```diff
@@ -38,7 +39,10 @@ const LIGHT_THEME = {
   inputBackground: INHERIT,
   inputForeground: INHERIT,
   inputBorder: 'var(--border)',
+  inputBorderRadius: '6px',
```

Comment:
let's move/rename this to `formContentBorderRadius` and apply it to inputs and submit/cancel buttons - from a design POV, the children elements inside of the form should have the same border radius